### DISCUSSION TITLE
Menu elements *jump* in Ikemen, they *scroll* instead in Mugen

### DISCUSSION THREAD

**Discussion by Kasasagi77** [ikemen-engine/Ikemen-GO#1378] [Feature Request] [answered]
### Describe the bug

When a **limited** number of menu items is set to display in a screenpack, the menu items **scroll smoothly** in the original Mugen engine, in Ikemen they simply **jump**.

### To Reproduce

1. Use a screenpack with a limited number of item visible.
2. Press down several times.
3. Compare visual effects between Mugen and Ikemen.

### Expected behavior

The menu items smoothly scroll.

### Engine Version (or source code date)

2023-08-23

### Operating system

Linux

**Comment by Kasasagi77** [comment]
Hello @K4thos.

From my understanding this is a very direct report of a difference in behaviour between Ikemen and Mugen, in the scope of screenpack effects.

Can I ask why it was demoted from an "Issue" to a "Discussion"?

**Comment by K4thos** [comment]
I don't think we should consider such cosmetic as a missing feature. Not an issue, imo.

**Comment by rakieldev** [comment, marked as answer]
Added by #2666
